dilqr_rs/dilqr.py

```diff
@@ -0,0 +1,168 @@
+import numpy as np
+import time
+
+from dilqr_rs.tv_dlqr import TV_DLQR, get_solver
+
+class DiLQR():
```

Comment:
Style: `DiLqr` instead of `DiLQR`, following the rules here: https://drake.mit.edu/styleguide/cppguide.html#General_Naming_Rules

Reply:
This is a tough one...., but I'll conform to camel case :P Agreed on `DiLqr`